hyperspectral/notebooks/spectra_demo.ipynb

```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demo of Spectrum and SpectralLibrary\n",
+    "\n",
+    "To run this demo, first download the complete ECOSTRESS spectral library data from the [ECOSTRESS website](https://speclib.jpl.nasa.gov/download) and unpack into a folder named `ECOSTRESS` in the current working directory.\n",
```

Comment:
I followed these instructions, and because I figured "why not?" I requested all spectra. After checking out I got an email that it could be up to _a week_ until I get my files. If you have files necessary to run the demo and the license of these files permits it, could we upload whatever you have to a private S3 bucket in the NASA account and point to that instead?

Reply:
I got that same email when I requested, and was served a response before EOD.  This might be a YMMV situation.  I can upload to an S3 bucket, though.

Reply:
👍 let's do that, so that Azavea devs don't have to repeat query JPL and we bring friction way down for dev/testing.

As an aside, I got my files this am!